perftracker/models/job.py

```diff
@@ -48,6 +48,8 @@ class JobModel(models.Model):
     regression      = models.ForeignKey('perftracker.RegressionModel', help_text="Regression of this job", related_name="regr", null=True, blank=True, on_delete=models.CASCADE)
     regression_tag  = models.CharField(max_length=512, help_text="MyProduct-2.0 regression", null=True, blank=True, db_index=True)
 
+    shown           = models.BooleanField(help_text="Show job in regression", db_index=True, default=True)
+
```

Comment:
Oh. Now I see you also have the  "shown" attribute.
1. The name is too vague. "shown" where?
2. Why is the 'regression' foreign key not enough?